src/main/java/com/bobi89/medicalclinic/controller/PatientController.java

```diff
@@ -54,5 +55,13 @@ public ChangePasswordCommand updatePatientPassword(@PathVariable String email,
         patientService.editPatientPassword(email, pass);
         return pass;
     }
+
+    @PostMapping("/{patientId}/{doctorId}/fitSlot")
```

Comment:
moim zdaniem ta ścieżka jest słaba przykładowe wywołanie takiego endpointu wyglądąłoby tak:

POST localhost:8080/patients/1/2/fitSlot <- czy to mi cos mowi? 

PATCH localhost:8080/visits/1 i w body id pacjenta
PATCH localhost:8080/visits/1/patient/1


ogólnie w sciezkach nie uzywamy camelCase tylko standardem są "-". 


ogólnie w sciezkach nie uzywamy camelCase tylko standardem są "-".